src/test/java/com/butbetter/storage/service/StorageServiceTests.java

```diff
@@ -0,0 +1,86 @@
+package com.butbetter.storage.service;
+
+import com.butbetter.storage.model.Address;
+import com.butbetter.storage.model.ProductInformation;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class StorageServiceTests {
+
+    private StorageService storageService;
+
+    @Autowired
+    private MockMvc mvc;
+
+    @BeforeEach
+    void setUp() {
+
+    }
+
+    @AfterEach
+    void tearDown() {
+        storageService = null;
+    }
+
+    @BeforeAll
+    static void beforeAll() {
+    }
+
+    @AfterAll
+    static void afterAll() {
```

Comment:
if not needed, it can be removed

Reply:
auto generated

Reply:
if you don't need it, you don't need to generate it